bin/create-files.js

```diff
@@ -33,12 +35,15 @@ const readAndWriteFiles = (projectDir) => {
         console.log("Created style.css")
     })
 }
-
+// Store the path name of the executable process

```

Comment:
This line is to get directory name from command line arguments,please change it accordingly